modules/data.atmosphere/R/half_hour_downscale.R

```diff
@@ -223,8 +223,16 @@ downscale_ShortWave_to_half_hrly <- function(df,lat, lon, hr = 0.5){
   
   for (k in 1:nrow(data.hrly)) {
     if(is.na(data.hrly$surface_downwelling_shortwave_flux_in_air[k])){
-      SWflux <- as.matrix(subset(df, .data$day == data.hrly$day[k] & .data$hour == data.hrly$hour[k], data.hrly$surface_downwelling_shortwave_flux_in_air[k]))
```

Comment:
I'm guessing a bit on what this was supposed to be, because it could not have worked as written -- not only is `.data` invalid inside base::subset (it's specific to tidy evaluation)  , but I'm not sure what subset would do with a bare number for its third argument. 

My replacement below seems reasonable if the goal of this step is "when SWflux is null in the hourly data, take it from the matching day and hour of the coarser-scale data", but please speak up if you see a better fix.